### PR TITLE
add interest and mental-health-recovery projects

### PR DESCRIPTION
**Pull request by parisa-zahedi**

### CODE REVIEW

File: docs/projects.md [hunk 1]
```diff
@@ -2,7 +2,23 @@
 
 Here you find an overview of our [current](#current-projects) and [completed projects](#completed-projects). According to Open Science principles, most of our software is publicly accessible on GitHub, and can be found via the title link.
 
+
 ### Current projects
+| Mental-health-recovery                                                                                                                                                                                       |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+| **Time frame**: 07/2024 - present                                                                                                                                                                            |
+| **Research domain**: ODISSEI Social Data Science Team (SoDa)                                                                                                                                                 |
```
Comment (qubixes) [inline docs/projects.md:10]: For homogeneity, I would put the Faculty of Social and Behavioural Sciences as the research domain. Then, you can mention the SoDa team in the description. Perhaps at some point we should extend the template to include researchers/collaborators.

**Review by qubixes — CHANGES_REQUESTED** [review]
Thanks for the update, see my comments for possible improvements.

File: docs/projects.md (hunk 1 above)
Comment (parisa-zahedi) [reply docs/projects.md]: thanks for the comment. Done!

File: docs/projects.md [hunk 2]
```diff
@@ -7,10 +7,10 @@ Here you find an overview of our [current](#current-projects) and [completed pro
 | Mental-health-recovery                                                                                                                                                                                       |
 |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (jelletreep) [inline docs/projects.md]:
```suggestion
| --- | 
```

File: docs/projects.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ Here you find an overview of our [current](#current-projects) and [completed pro
 | Mental-health-recovery                                                                                                                                                                                       |
```
Comment (jelletreep) [inline docs/projects.md]:
```suggestion
| Mental-health-recovery |                                                                                                                                                                            
```

**Review by jelletreep — APPROVED** [review]
Looks good! I removed some whitespace and '-' so the entries are similarly formatted to the other ones

**Pull request merged**